internal/command/command.go

```diff
@@ -21,6 +24,11 @@ func New() *Command {
 		utils.Fatal(err)
 	}
 
+	if flags.Version {
+		fmt.Println("Icinga DB version:", internal.Version)
+		os.Exit(0)
```

Comment:
In our `main.go` we have two constants for the exit code. The logging PR also calls `os.Exit()` outside of main after a fatal error has been logged. I think we should do this consistently and either use these constants everywhere or drop them. I opt for moving the constants to a new package, e.g. `exitcode`. @julianbrost Since you introduced these constants, what's your opinion here?

Reply:
I added them because otherwise `int run()` would have magic `return 0;` statements without an indication what this number is supposed to mean. They were inspired by the `EXIT_SUCCESS`/`EXIT_FAILURE` constants you have in C.

So from this point of view, just using `os.Exit(0)` here is fine, as it's pretty clear that it's an exit code.

Yet another possibility would be to have an `type ExitCode int` and use `return ExitCode(0);` instead.

I don't have a strong opinion towards any of these options, but I'd prefer one where there are `return 0;` statements without any indication that it's an exit code.

Reply:
I see. Makes sense!